### ISSUE TITLE
is_live problem

### ISSUE THREAD

**Issue by Multivalence** [kevinzg/facebook-scraper#300]
the is_live key you receive after getting an accounts posts, is always False even if the post is live.

**Comment by Multivalence** [comment]
I did a little bit of digging and I found that the headers return "was live" even though they are currently live. The regex is looking for "is live" which is probably why it says False.

**Comment by neon-ninja** [comment]
This is working fine for me, can you post a link / screenshot of a sample live post you're having this problem with? I tested with https://m.facebook.com/230809307041021/videos/3022632247966470 and it correctly says "is live"

**Comment by Multivalence** [comment]
One profile this issue frequently happens to is https://www.facebook.com/versatileebby. The post in question is this one but you probably are gonna have to wait until they go live again: https://www.facebook.com/versatileebby/videos/313628186955399/

**Comment by neon-ninja** [comment]
Says "is live now" for me, do you see something different?

**Comment by neon-ninja** [comment]
They're live now at https://www.facebook.com/versatileebby/videos/756910395002250:

```python
[{'available': True,
  'comments': 95,
  'comments_full': None,
  'factcheck': None,
  'image': None,
  'image_id': None,
  'image_ids': [],
  'image_lowquality': None,
  'images': [],
  'images_description': [],
  'images_lowquality': [],
  'images_lowquality_description': [],
  'is_live': True,
  'likes': 9168,
  'link': 'https://www.instagram.com/versatileebby',
  'original_request_url': 'https://m.facebook.com/756910395002250',
  'post_id': '756910395002250',
  'post_text': '🔞 18+ GTA 5 Mods\n'
               'Want to Become my supporter -\n'
               'www.facebook.com/becomesupporter/2202626363188097/\n'
               'Instagram - www.instagram.com/versatileebby\n'
               'Follow me in YouTube - www.youtube.com/c/Versatileebby\n'
               'Guys join Versatile Family 🔥- '
               'https://www.facebook.com/groups/358141148917890',
  'post_url': 'https://facebook.com/story.php?story_fbid=756910395002250&id=2202626363188097',
  'reaction_count': None,
  'reactions': None,
  'reactors': None,
  'shared_post_id': None,
  'shared_post_url': None,
  'shared_text': '',
  'shared_time': None,
  'shared_user_id': None,
  'shared_username': None,
  'shares': 1036,
  'text': '🔞 18+ GTA 5 Mods\n'
          'Want to Become my supporter -\n'
          'www.facebook.com/becomesupporter/2202626363188097/\n'
          'Instagram - www.instagram.com/versatileebby\n'
          'Follow me in YouTube - www.youtube.com/c/Versatileebby\n'
          'Guys join Versatile Family 🔥- '
          'https://www.facebook.com/groups/358141148917890',
  'time': datetime.datetime(2021, 6, 1, 9, 52, 48),
  'user_id': '2202626363188097',
  'user_url': 'https://facebook.com/versatileebby/?refid=52&__tn__=C-R',
  'username': 'Versatile',
  'video': 'https://video.fhlz2-1.fna.fbcdn.net/hvideo-frc-ftw/s1002/_nc_cat-1/v/rziOOJhMtppCv_nNed-c5/_nc_ohc-WIXQwP_tEZYAX_aHhf-/live-dash/dash-abr5/3973927289391320.mpd?_nc_rl=AfAoayewtf9_CdCbORI&ccb=2-4&ms=m_CNP&oh=5604c33cec38e05f9129954f091c8404&oe=60B6F0F8',
  'video_duration_seconds': None,
  'video_height': 360,
  'video_id': '756910395002250',
  'video_quality': None,
  'video_size_MB': None,
  'video_thumbnail': 'https://scontent.fhlz2-1.fna.fbcdn.net/v/t15.5256-10/cp0/e15/q65/s851x315/186764658_756910841668872_2647637009409352198_n.jpg?_nc_cat=1&ccb=1-3&_nc_sid=ccf8b3&_nc_ohc=VWKfoXOnNZQAX9scpKt&_nc_ht=scontent.fhlz2-1.fna&tp=9&oh=075849efa3447b09d267c41a605fda6c&oe=60BA684E',
  'video_watches': 0,
  'video_width': 360,
  'w3_fb_url': None}]
```

**Comment by Multivalence** [comment]
I feel like I should reiterate my problem. 
This code:
```py
for post in get_posts(username, pages=1):
    print(post)
```
skips over posts (most commonly live ones) and goes to the next ones.

Heres an example: 

- Im trying to get a post from this account: https://www.facebook.com/TheMilianaire/

- This is what the text says on the most recent post: https://imgur.com/1KcLIOS (video link: https://www.facebook.com/TheMilianaire/videos/908247076685829/)

- This is what I get when I execute the code above (headers in extractors.py are being printed as well): https://imgur.com/fC7XioF

- Im assuming that these headers are coming from the older posts: https://imgur.com/WMEfwuh & https://imgur.com/OG1Ewlq

Hopefully that makes a bit more sense.

**Comment by neon-ninja** [comment]
I see - if I go to https://www.facebook.com/versatileebby, I see a livestream as the first post, but if I go to https://m.facebook.com/versatileebby/, it's missing some of the time, but depending on when I refresh the page, it might be shown. It's always there on mobile/mobile emulation. Not really sure why, but this code seems to fix it:

```python
print(next(get_posts("versatileebby", cookies="cookies.txt", pages=1))["is_live"])
_scraper.session.headers.update({
    "sec-fetch-user": "?1",
    "user-agent": "Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Mobile Safari/537.36"
})
print(next(get_posts("versatileebby", cookies="cookies.txt", pages=1))["is_live"])
```
outputs 
```python
False
True
```

**Comment by Multivalence** [comment]
Seems to be working now. Thank you! Will this change be published to PyPI?

**Comment by neon-ninja** [comment]
Sure, published as 0.2.41

**Comment by Multivalence** [comment]
Thank you. Great job on the scraper btw. Love it!
